examples/grid2grid/main.py

```diff
@@ -0,0 +1,161 @@
+from evasdk import Eva
+from evaUtilities import *
+from progress.bar import *
+from config.config_manager import load_use_case_config
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class EvaGrids:
+    def __init__(self, _eva, _plot_on_off):
+        names = config['grids']['names']
+        self.eva = _eva
+        self.plot = _plot_on_off
+        self.counter_max = {names[0]: config['grids']['row'][names[0]]*config['grids']['col'][names[0]]-1,
+                            names[1]: config['grids']['row'][names[1]]*config['grids']['col'][names[1]]-1}
+        # Check that slots in second grid are >= than first grid
+        slots = [abs(config['grids']['row'][names[0]]) * abs(config['grids']['col'][names[0]]),
+                 abs(config['grids']['row'][names[1]]) * abs(config['grids']['col'][names[1]])]
+        if slots[0] > slots[1]:
+            raise Exception('Drop-off grid is smaller than pick-up grid')
+
+    @staticmethod
+    def _create_grid(obj):

```

Comment:
obj is a non descriptive name, what is obj?

Reply:
obj was the i-th grid. Changed this to a more descriptive grid_iter